niworkflows/interfaces/morphology.py

```diff
@@ -0,0 +1,105 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Copyright 2021 The NiPreps Developers <nipreps@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# We support and encourage derived works from this project, please read
+# about our expectations at
+#
+#     https://www.nipreps.org/community/licensing/
+#
+""" Handling brain mask"""
+
+from nipype.interfaces.base import (
+    traits,
+    TraitedSpec,
+    BaseInterfaceInputSpec,
+    File,
+    SimpleInterface,
+)
+
+
+class _CrownMaskInputSpec(BaseInterfaceInputSpec):
+    in_segm = File(
+        exists=True, mandatory=True, position=0, desc="Input discrete segmentation of the brain."
+    )
+    in_brainmask = File(exists=True, mandatory=True, position=1, desc="Brain mask.")
+    radius = traits.Int(default_value=2, usedefault=True, desc="Radius of dilation")
+
+
+class _CrownMaskOutputSpec(TraitedSpec):
+    out_mask = File(exists=False, desc="Crown mask")
+
+
+class CrownMask(SimpleInterface):
```

Comment:
```suggestion
class BinaryDilation(SimpleInterface):
```

Reply:
I have some doubt about the name BinaryDilation. This interface do not return the dilated mask, but the dilated mask - the original mask.

Reply:
Yup, see all my comments. The plan is actually change that and make the interface more general. The subtraction does not happen here anymore.